### ISSUE TITLE
Implement first iteration of Dropdown Menu + Menu Item

### ISSUE THREAD

**Issue by trybern** [FHIDev/Fhi.Designsystem#250]
### Acceptance criteria

- It should be implemented in accordance [with the designs in Figma](https://www.figma.com/design/VgQG6XeEbFOmHIrxyAOACR/FHI-Designsystem?node-id=10431-2537&p=f&t=BiVDaU6FU4x2xarg-11)
- It should be accessable from the NPM package and CDN
- It should have automated unit and accessibility tests with appropriate coverage
- It should have a story with an introduction, usage instructions, implementation examples and action interactivity
- It should conform to the Web standard on inputs/outputs (e.g form interactions)
- It should have appropriate component level design tokens that references semantic, and if appropriate, primitive level tokens.

<img width="962" height="530" alt="Image" src="https://github.com/user-attachments/assets/8fd321cb-b82f-46f7-96fd-f9b8f36eb5ad" />

### Additional information

- [Building like it's 1984: A comprehensive guide to creating intuitive context menus](https://height.app/blog/guide-to-build-context-menus)
